promql/testdata/operators.test

```diff
@@ -35,6 +35,16 @@ eval instant at 50m - - - SUM(http_requests) BY (job)
 eval instant at 50m - - - 1
   -1
 
+eval instant at 50m -2^---1*3
+  -0.75
+
+eval instant at 50m 2/-2^---1*3+2
+  5
+
+eval instant at 50m -10^3 * - SUM(http_requests) BY (job) ^ -1
```

Comment:
This doesn't seem right, `-x ^ -1` should be same as `-1 / x `

Reply:
It is.

You can get to this expression by transforming the below expression:

```
    1000 / SUM(http_requests) BY (job)
=  - 1000 / - SUM(http_requests) BY (job)            
=  - 10^3 / - SUM(http_requests) BY (job)
=  - 10^3 * -1 / SUM(http_requests) BY (job)        | -1 / x = -x ^ -1
=  - 10^3 * - SUM(http_requests) BY (job) ^ -1
```